Sources/Vaporiser/Router.swift

```diff
@@ -150,11 +152,22 @@ private extension String {
 
         return match
     }
+
+    func matches(queryParameters: [String: String]?) -> Bool {
```

Comment:
@ericgimenezgalera Can't we use a statically typed `URLQueryParams` or another wrapper instead of passing a dictionary?